### PR TITLE
LabHub: Remove redundant teams list

### PR DESCRIPTION
**Pull request by nvzard**
Redefine excessive usage for the list
of teams.

Closes https://github.com/coala/corobo/issues/575

### CODE REVIEW

**Comment by nvzard** [comment]
Unblocks https://github.com/coala/corobo/pull/552

File: plugins/labhub.py [hunk 1]
```diff
@@ -71,6 +71,17 @@ def TEAMS(self):
     def TEAMS(self, new):
         self._teams = new
 
+    def team_mapping(self):
+        return {
+            'newcomers': self.TEAMS[self.GH_ORG_NAME + ' newcomers'],
+            'developers': self.TEAMS[self.GH_ORG_NAME + ' developers'],
+            'maintainers': self.TEAMS[self.GH_ORG_NAME + ' maintainers']
```
Comment (jayvdb) [inline plugins/labhub.py]: trailing comma.

File: tests/labhub_test.py [hunk 2]
```diff
@@ -111,6 +111,8 @@ def test_is_room_member(self):
     def test_hello_world_callback(self):
         teams = {
             'coala newcomers': self.mock_team,
+            'coala developers': self.mock_team,
+            'coala maintainers': self.mock_team
```
Comment (jayvdb) [inline tests/labhub_test.py]: trailing comma

File: plugins/labhub.py [hunk 3]
```diff
@@ -89,40 +100,28 @@ def invite_cmd(self, msg, match):
...
 
         self.log.info('{} invited {} to {}'.format(inviter, invitee, team))
 
-        valid_teams = ['newcomers', 'developers', 'maintainers']
-        if team not in valid_teams:
+        if team not in self.team_mapping():
             return 'Please select from one of the valid teams: ' + ', '.join(
-                    valid_teams)
-
-        def invite(invitee, team):
```
Comment (jayvdb) [inline plugins/labhub.py]: keep the function, change what it does.
that is less disruptive to existing code, making it easier to read&review the changes

File: plugins/labhub.py [hunk 4]
```diff
@@ -89,40 +100,28 @@ def invite_cmd(self, msg, match):
...
-        is_developer = self.TEAMS[self.GH_ORG_NAME +
-                                  ' developers'].is_member(inviter)
-        is_maintainer = self.TEAMS[self.GH_ORG_NAME +
-                                   ' maintainers'].is_member(inviter)
+        is_developer = self.is_team_member(inviter, 'developers')
+        is_maintainer = self.is_team_member(inviter, 'maintainers')
 
         self.log.info('{} invited {} to {}'.format(inviter, invitee, team))
 
-        valid_teams = ['newcomers', 'developers', 'maintainers']
```
Comment (jayvdb) [inline plugins/labhub.py]: `valid_teams = elf.team_mapping()` will require less changes to existing lines, and be more efficient.

**Comment by nvzard** [comment]
Is there something wrong with @gitmate-bot?

**Comment by jayvdb** [comment]
ack https://github.com/coala/corobo/pull/576/commits/de4fbaff1bd227d078f15e8221ea4941fdcf4450

**Comment by jayvdb** [comment]
@gitmate-bot ff

**Pull request merged**